apps/dashboard/lib/dashboard/component.ex

```diff
@@ -1,19 +1,23 @@
 defmodule Dashboard.Component do
   defmacro __using__(_opts) do
     quote do
-      use GenServer
+      use GenServer, restart: :transient
```

Comment:
Before I added this, the GenServer would shut down and then I'd start to see API requests again. I thought to myself, "it's like something in the code is restarting the server once it terminates." In fact, it was doing exactly what a good Supervisor _should_ do! 😆 

The default is `restart: :permanent`, which will always restart it. `restart: :transient` will only restart it if its shutdown was abnormal.

Reply:
https://www.youtube.com/watch?v=4SCwubzqsVU

Reply:
That made me LOL